packages/core/src/browser/style/view-container.css

```diff
@@ -75,10 +75,19 @@
 }
 
 .theia-view-container .part > .header .label {
+    flex: 0;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+}
+
+.theia-view-container .part > .header .subheading {
     flex: 1;
     overflow: hidden;
     white-space: nowrap;
     text-overflow: ellipsis;
+    padding-left: 12px;
```

Comment:
We should compute this value I believe rather than hardcoding.
Example:

https://github.com/eclipse-theia/theia/blob/a6422c39d6c09095f2bdc8cb4af4427e6bc78082/packages/debug/src/browser/style/index.css#L142

Reply:
Great suggestion, thanks!